css/style.css

```diff
@@ -114,6 +119,220 @@ body {
 
 /* END */
 
+/* WORKS SECTION */
+.work__section {
+  margin-top: 114px;
+  display: grid;
+  row-gap: 88px;
+}
+
+.work__description {
+  display: flex;
+  flex-direction: column;
+  max-width: 327px;
+  margin: 0 auto;
+  border: 2px solid var(--borderColor);
```

Comment:
[Optional] Great job on the border implementation! I noticed that the border you've applied looks good. However, according to the design specifications, it should be a 1px border instead of a 2px border.

To align your implementation with the design, I recommend updating the border width to 1px. 

![image](https://github.com/Feliperto/PortfolioWebsite-Microverse/assets/55840999/7a363587-c1e0-402c-870d-8390f584c80d)